pkg/cmd/pr/checkout/checkout.go

```diff
@@ -33,6 +33,8 @@ type CheckoutOptions struct {
 	RecurseSubmodules bool
 	Force             bool
 	Detach            bool
+	// Local is the branch name to use when renaming the checked out branch.
+	Local string
```

Comment:
Could this be renamed to `BranchName` so that the comment explaining this field is not needed anymore?

Reply:
How do you feel about `LocalBranch` instead? I think `BranchName` might get confusing since there's also a `Branch` field too.

Reply:
I could also change `Branch` to `RemoteBranch` as well, so there'd be `LocalBranch` and `RemoteBranch`.

Reply:
BranchName is fine. It also matches the name of the cli flag, so it will be recognizable. The types of `Branch` and `BranchName` fields are different, so there should be no confusion.